operations/helm/charts/mimir-distributed/templates/query-scheduler/query-scheduler-dep.yaml

```diff
@@ -44,8 +44,6 @@ spec:
             - "-target=query-scheduler"
             - "-config.expand-env=true"
             - "-config.file=/etc/mimir/mimir.yaml"
-            - "-server.grpc.keepalive.max-connection-age=2562047h" # 100000 days, effectively infinity
```

Comment:
The PR description doesn't say why #3262 is reverted here.

Reply:
It seems that the timeline for these changes was the following:

1. In #2862 we set `grpc_server_max_connection_age: 2m` for all gRPC servers (i.e. updated common `mimir.yaml` for all modules).
1. This caused issues in some modules, including "query-scheduler".
2. In #3262 we hardcoded the `grpc_server_max_connection_age` via the scheduler's flags, to overwrite changes in (1)

This PR, effectively rolls back parts of the original changes from #2862, so the connection-age in "query-scheduler" should be set to defaults "MaxInt32", matching to what jsonnet has. This makes #3262 redundant.

Reply:
Agree with @narqo .